src/scripts/nomenclatures/plain/membershipState.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  active: 'MEMBERSHIP_ACTIVE',
+  expired: 'MEMBERSHIP_EXPIRED'
```

Comment:
I think you need to add `MEMBERSHIP_ALL` so user can disable filtering by membership, once it's selected